docs/sql-ref-ansi-compliance.md

```diff
@@ -66,11 +66,9 @@ SELECT abs(-2147483648);
 +----------------+
 ```
 
-### Type Conversion
+### Cast
```

Comment:
In this section, it says `In future releases, the behaviour of type coercion might change along with the other two type conversion rules.`

We should update it.